test/others/criu-ns/run.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+
+import multiprocessing
+import os
+import pty
+import shutil
+import subprocess
+import sys
+import time
+
+
+CRIU_BIN = "../../../criu/criu"
+CRIU_NS = "../../../scripts/criu-ns"
+
+
+def check_dumpdir():
+    if os.path.isdir("dumpdir"):
+        shutil.rmtree("dumpdir")
+    os.mkdir("dumpdir", 0o755)
+
+
+def create_pty():
+    fd_m, fd_s = pty.openpty()
+    return (os.fdopen(fd_m, "wb"), os.fdopen(fd_s, "wb"))
+
+
+def test_dump_and_restore_with_shell_job():
+    check_dumpdir()
+    os.setsid()
+
+    with open("running", "w") as file:
+        pass
+    pid = os.fork()
+    if pid == 0:
+        while True:
+            if not os.access("running", os.F_OK):
+                sys.exit(0)
+            time.sleep(1)
+
+    cmd = [CRIU_NS, "dump", "-D", "dumpdir", "-v", "--shell-job",
```

Comment:
```Suggestion
    cmd = [CRIU_NS, "dump", "-D", "dumpdir", "-v4", "-o", "dump.log", "--shell-job",
```

Reply:
If I add the `v4` and `o` options to CRIU dump and restore in this test script, then the CI `stdout` will be empty when the actual dump/restore (multiple times) is occurring. If someone is watching CI output in realtime, this could be somewhat confusing. What do you think @Snorch

Reply:
> If I add the v4 and o options to CRIU dump and restore in this test script, then the CI stdout will be empty when the actual dump/restore (multiple times) is occurring. 

@warusadura `-v4` increases the verbosity level to include debug logs. This is particularly useful for debugging when a test fails in CI. However, this would also increase the amount of logs. Thus, it would be good to read the content of the log file only when necessary (as suggested in https://github.com/checkpoint-restore/criu/pull/2114#discussion_r1161260946).

Reply:
understood, thanks!

Reply:
updated all the `criu-ns` commands with `-v4` option. But, I like to keep the realtime dump/restore log in the CI stdout instead of a log file :)